### PR TITLE
ADCM-3182 Download task log archive

### PR DESCRIPTION
**Pull request by winmasta**

### CODE REVIEW

**Comment by winmasta** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @winmasta! Build has been launched!

File: python/api/job/views.py [hunk 1]
```diff
@@ -31,202 +39,300 @@
...
+            if directory.is_dir():
+                for log_file in Path(settings.RUN_DIR, str(job.pk)).iterdir():
+                    tarinfo = tarfile.TarInfo(f"{job.pk}-{dir_name_suffix}/{log_file.name}")
+                    tarinfo.size = log_file.stat().st_size
+                    tar_file.addfile(tarinfo=tarinfo, fileobj=io.BytesIO(log_file.read_bytes()))
+            else:
+                log_storages = LogStorage.objects.filter(job=job, type__in={"stdout", "stderr"})
+                for log_storage in log_storages:
+                    tarinfo = tarfile.TarInfo(
+                        f"{job.pk}-{dir_name_suffix}/ansible-{log_storage.type}.txt"
```
Comment (a-alferov) [inline python/api/job/views.py:151]:
```suggestion
                        f"{job.pk}-{dir_name_suffix}/{log_storage.name}-{log_storage.type}.txt"
```

File: python/cm/tests/files/task_log_download/1/ansible.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"ansible":  "json", "job_id": 1}
```
Comment (a-alferov) [inline python/cm/tests/files/task_log_download/1/ansible.json:1]: Pls rename files to `config.json`

**Review by a-alferov — CHANGES_REQUESTED** [review]

File: python/api/job/views.py (hunk 1 above)
Comment (winmasta) [reply python/api/job/views.py]: Fixed

File: python/cm/tests/files/task_log_download/1/ansible.json (hunk 2 above)
Comment (winmasta) [reply python/cm/tests/files/task_log_download/1/ansible.json]: Fixed

**Review by a-alferov — APPROVED** [review]

**Pull request merged**